man/systemd.network.xml

```diff
@@ -2257,6 +2257,53 @@
       </variablelist>
   </refsect1>
 
+  <refsect1>
+    <title>[TrafficControlQueueingDiscipline] Section Options</title>
+    <para>The <literal>[TrafficControlQueueingDiscipline]</literal> section manages the Traffic control. It can be used
+    to configure the kernel packet scheduler and simulate packet delay and loss for UDP or TCP applications,
+    or limit the bandwidth usage of a particular service to simulate internet connections.</para>
+
+    <variablelist class='network-directives'>
+      <varlistentry>
+        <term><varname>Parent=</varname></term>
+        <listitem>
+          <para>Specifies Queueing Discipline (qdisc). Takes one of <literal>root</literal> or <literal>clsact</literal>.
+          Defaults to <literal>root</literal>.</para>
+        </listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>NetworkDelaySec=</varname></term>
+        <listitem>
+          <para>Specifies the network fixed amount of delay to all packets going out of the interface. Defaults to unset.</para>
+        </listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>NetworkDelayJitterSec=</varname></term>
+        <listitem>
+          <para>Specifies the random delay to be added to <varname>NetworkDelaySec=</varname>. Defaults to unset.</para>
+        </listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>NetworkDelayLimitPacket=</varname></term>
```

Comment:
This setting has nothing to do with delay... I suggest something like `PacketLimit=` or `MaxPacketCount=`

Reply:
Ditto

Reply:
What is the "ditto" about here? my comment was mainly about the fact that "delay" and "limit" are independent from one another. So I'm suggesting to drop "delay".

Reply:
See my comment about emulator.